src/auro-popover.js

```diff
@@ -16,17 +16,22 @@ import Popover from "./popover";
  *
  * @attr {String} placement - Expects top/bottom - position for popover in relation to the element.
  * @attr {String} for - Defines an `id` for an element in the DOM to trigger on hover/blur.
+ * @attr {boolean} isPopoverVisible - Boolean for if popover is visible or not.
+ * @attr {boolean} isSticky - If true, when user clicks trigger, the popover will persist its visibility. If false, popover will disappear when mouseout over trigger.
  * @slot - Default unnamed slot for the use of popover content
  * @slot trigger - Slot for entering the trigger element into the scope of the shadow DOM
  */
 class AuroPopover extends LitElement {
   constructor() {
     super();
     this.placement = 'top';
+    this.isPopoverVisible = false;
```

Comment:
This is part of the private API and should not be exposed in the docs. 

See this code as an example of how to remove this from the generated API doc
https://github.com/AlaskaAirlines/auro-icon/blob/master/src/auro-icon.js

Reply:
Line has been changed according to the example provided.